### PR TITLE
Finish GitHub actions transition

### PR DESCRIPTION
**Pull request by pedrofurtado**
Hello, @unixmonkey ! 👋 

This PR finishes the Github Actions transition started at #986 🎉 

New cops are present in recent rubocops versions (that is needed for Ruby 3 support), but initially them are disabled. Later, it will be possible to "turn on" each one of them and change the source code to be compliant with each rule. But is better to make it gradually 🤝 

Let me know your thoughts about it 🕺 

Thanks! 🤓

### CODE REVIEW

File: wicked_pdf.gemspec [hunk 1]
```diff
@@ -21,7 +21,7 @@ DESC
     'changelog_uri' => 'https://github.com/mileszs/wicked_pdf/blob/master/CHANGELOG.md'
   }
 
-  spec.required_ruby_version = Gem::Requirement.new('>= 2.2')
+  spec.required_ruby_version = Gem::Requirement.new('>= 2.6')
```
Comment (unixmonkey) [inline wicked_pdf.gemspec:24]: Is there a reason to raise this minimum Ruby version, and by so much? I'd like people on older rubies to still be able to use this gem.

Comment (pedrofurtado) [reply wicked_pdf.gemspec:24]: We can maintain this old ruby version, if you prefer, no problem 🤝 But CI tests may not guarantee it works well on such older versions of the gem

Comment (pedrofurtado) [reply wicked_pdf.gemspec]: I made rollback on this change, returning to ruby 2.2 👍

Comment (mathieujobin) [reply wicked_pdf.gemspec]: My two cents here, I usually keep CI to run for old versions as long as tests pass.
Once we hit a failure, then we take the decision if it's worth fixing or moving on, deprecating the failing version.

File: gemfiles/rails_edge.gemfile [hunk 2]
```diff
@@ -1,9 +0,0 @@
-source 'https://rubygems.org'
-
-gem 'bootsnap' # required to run `rake test` in Rails 6.0
-gem 'mocha', '= 1.3' # newer versions blow up
-gem 'rails', git: 'https://github.com/rails/rails.git'
-gem 'rdoc'
-gem 'sqlite3', '~> 1.4'
-
-gemspec path: '../'
```
Comment (mathieujobin) [inline gemfiles/rails_edge.gemfile:9]: Why not keeping Rails Edge? 7.0 is just around the corner. I have an app running it already.
Maybe just allow-failures on edge

**Pull request merged**